src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/matmul_weights_decompression.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#ifdef CPU_DEBUG_CAPS
```

Comment:
Will these test instances work correctly if `OV_CPU_ENABLE_DNNL_MAMTUL_FOR_FC` is not set? I am a bit confused by the fact that we will have red tests in case of some specific CMAKE options enabled

Reply:
They won't. Good point.
I refactored these instances in https://github.com/openvinotoolkit/openvino/pull/27459/commits/51b67cac8980e4c8f9cf60a357547563e774151a. Now test suites are enabled in default build. The difference is expected decompression usage based on debug env variable